ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkReader.java

```diff
@@ -390,9 +390,9 @@ private Void readSvc(String[] tokens) {
 
 
         Terminal t = svc.getTerminal();
-        t.setP(Float.NaN).setQ(q);
-
-        svc.setVoltageSetPoint(targetV);
+        t.setP(0).setQ(q);
```

Comment:
Seems inconsistent with the IIDM parser: if not set, P is equals to NaN. It's the reason why we ask in the previous commit to set P to NaN here.